packages/cli/src/commands/init/utils/repo.spec.ts

```diff
@@ -1,14 +1,36 @@
 import { configureGitSetup, logFailure, logSuccess, selectRepo } from './repo';
 
 describe('Repo', () => {
-  it('clone should throw an error and call removeDir', async () => {
+
```

Comment:
Does the change in this test cover all your new additions?

Reply:
No, I wasn't sure how to approach the last 2 functions of removing git and then init git. 
I was thinking this is something that could be capture with end to end testing.